src/query/service/src/interpreters/interpreter_copy.rs

```diff
@@ -254,67 +240,120 @@ impl CopyInterpreter {
         &self,
         plan: &CopyIntoTablePlan,
     ) -> Result<PipelineBuildResult> {
+        let catalog = plan.catalog_name.as_str();
+        let database = plan.database_name.as_str();
+        let table = plan.table_name.as_str();
+
         let ctx = self.ctx.clone();
-        let to_table = ctx
-            .get_table(&plan.catalog_name, &plan.database_name, &plan.table_name)
-            .await?;
+        let to_table = ctx.get_table(catalog, database, table).await?;
 
-        let (mut build_res, source_schema, files) = if let Some(query) = &plan.query {
-            let (build_res, source_schema) = self.build_query(query).await?;
-            (
-                build_res,
-                source_schema,
-                plan.stage_table_info
+        let mut build_res;
```

Comment:
I have careful review these changed codes, the logic is no changed, but some stateful test failed with: 
`+ERROR 1105 (HY000) at line 1: Internal. Code: 1001, Text = Cannot resize empty pipe..`
@JackTan25

Reply:
https://github.com/datafuselabs/databend/actions/runs/5529492495/jobs/10087872738?pr=12062#step:4:454

Reply:
https://github.com/datafuselabs/databend/pull/12062/files#r1260972866